src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigRunners.cs

```diff
@@ -40,10 +34,66 @@ public static void Run(ConfigPathsArgs args, Func<ILogger> getLogger)
         }
     }
 
+    internal static class ConfigGetRunner
+    {
+        public static void Run(ConfigGetArgs args, Func<ILogger> getLogger)
+        {
+            RunnerHelper.ValidateArguments(args, getLogger);
+
+            if (args.AllOrConfigKey == null)
+            {
+                throw new CommandException(string.Format(CultureInfo.CurrentCulture, Strings.ConfigCommandKeyNotFound, args.AllOrConfigKey));
+            }
+
+            var settings = RunnerHelper.GetSettingsFromDirectory(args.WorkingDirectory) as Settings;
+            if (settings == null)
+            {
+                return;
+            }
+            ILogger logger = getLogger();
+
+            if (args.AllOrConfigKey.Equals("all", StringComparison.OrdinalIgnoreCase))
+            {
+                var sections = settings.ComputedSections;
```

Comment:
Can this call GetSection instead of directly accessing the internal dictionary? This would also prevent us from having to expose that publicly.

Reply:
Initially, that was my plan, but since I don't know which sections exist and I want all the sections in the settings, using `ComputedSections` seemed like a better option.